src/components/button/button.stories.js

```diff
@@ -1,32 +1,105 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { storiesOf } from '@storybook/react';
 import { text, select, boolean } from '@storybook/addon-knobs';
 import { action } from '@storybook/addon-actions';
 import OptionsHelper from '../../utils/helpers/options-helper';
-import notes from './notes.md';
-import Button from './button';
+import notes from './documentation/notes.md';
+import Info from './documentation/Info';
```

Comment:
in `documentation` we can create an `index.js` file and export both: `notes` and `info `so we can just import everything from one source
```js
//index.js
export { default as notes } from './notes.md'
export { default as info} from './info'
```
then we can just import 
```js
import { notes, info } from './documentation'
```